### PR TITLE
Home Page

### PR DESCRIPTION
**Pull request by i-am-a-codegirl**
Please review my Home Page of component library 

Live Link: https://css-companion.netlify.app/ 

This page contains 

- Nav: Home components and git
- Main: Introduction Text and link to components page
- Footer: contains links to all social sites.

image
image

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+body{
+font-family: 'Roboto', sans-serif;
+box-sizing: border-box;
+margin: 0px;
+padding: 0px;
+background-color: #111827;
+}
+
+* {
+
```
Comment (khushijohri2001) [inline style.css]: you can remove extra space here

File: style.css [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+    " footer footer footer";
+}
+
+nav{
+    display: flex;
+    gap: 20px;
+    width: 100vw;
+    flex-direction: column;
+    grid-area: nav;
+
```
Comment (khushijohri2001) [inline style.css]: here too

**Review by khushijohri2001 — CHANGES_REQUESTED** [review]
You can use the Prettier tool for your code if you use VS code and make the alignment and spacing in your code even. This will make it look consistent and clean

File: index.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        <h2 class="imp-text">CSS-Companion</h2>
+        <ul>
+            <li><a class="current-pg" href="/">Home</a></li>
+            <li><a href="../component/avatar/component.html">Components-docs</a></li>
+            <li><a href="/">git</a></li>
+        </ul>
+    </nav>
+        <main>
+            <h3 class="imp-text"><span class="impact">for - </span>your beautiful websites & beuatiful you</h3>
+            <p> Hy! there
```
Comment (samflab) [inline index.html:25]: this will be hey there not hy there

File: style.css [hunk 4]
```diff
@@ -0,0 +1,263 @@
...
+    }
+    
+ }
+
+
+ 
+ @media (max-width: 1280px){
+
+    html{
+        font-size: 92%;
```
Comment (samflab) [inline style.css:171]: we don't use % for font-size, its a wrong convention, use `rem` or `em` or `px` instead.

**Review by samflab — APPROVED** [review]

**Comment by samflab** [comment]
Everything else looks good to merge, you can make these changes later.

**Pull request merged**